src/lib.rs

```diff
@@ -153,8 +159,8 @@ impl<'py> Serializer for Pythonizer<'py> {
     fn serialize_f64(self, v: f64) -> Result<Self::Ok, Self::Error> {
         Ok(v.into_py(self.py))
     }
-    fn serialize_char(self, _v: char) -> Result<Self::Ok, Self::Error> {
-        todo!()
+    fn serialize_char(self, v: char) -> Result<Self::Ok, Self::Error> {
+        Ok(v.to_string().into_py(self.py))
```

Comment:
I went for a `str` representation here since this is a unicode character and not just bytes.

Reply:
That sounds reasonable. One other possible design would be to serialize the same as `u32` and output a scalar value. I wasn't sure what would be more useful for users, so happy to listen to your opinion on this.

(On the python side the code could then use the `chr()` builtin if they want to convert the scalar value to a `str`.)